### ISSUE TITLE
Fuelphp simplified Chinese Manual － FuelPHP简体中文手册

### ISSUE THREAD

**Issue by wxb** [fuel/docs#703]
@stevewest @sagikazarmark 
## simplified Chinese Manual

Hi, I translated this site [www.fuelphp.com](http://www.fuelphp.com/) into Chinese (Simplified), Can you create a link in Chinese (Simplified) , This can help Chinese developers
## 简体中文手册

本手册是本人在使用fuelPHP中，由于原有fuelPHP官网使用了一些google的jQuery，导致网页加载慢甚至无法加载。自己download了官网， 修改了其中jQuery加载的url，同时针对繁体中文对于很多人来说阅读还是有一些费劲，所以利用业余时间逐步翻译成简体中文。分享给大家， 希望可以帮到想要了解fuelPHP的小伙伴么！
## 手册地址

[中文手册](http://wxb.github.io/fuelPHP.zh-CN/index.html)
[中文手册GitHub](https://github.com/wxb/fuelPHP.zh-CN)
## 状态

简直中文正在翻译中，欢迎对fuelphp感兴趣的小伙伴加入 :smile:

**Comment by WanWizard** [comment]
We can add the link to this page: http://docs.fuelphp.com/installation/tutorials.html as it is external information.

**Comment by WanWizard** [comment]
@kenjis do you know if docs are available translated into Japanse? We could add that too...

**Comment by kenjis** [comment]
@WanWizard 
http://fuelphp.jp/

In Japanese FuelPHP community, we've been translating the documentation since 2011, but it is not finished yet.
However now we reach very close to the end. I think we could finish 1.8-dev translation in this month.

**Comment by WanWizard** [comment]
Would be a good idea to add this link to the docs all well?

**Comment by kenjis** [comment]
Yes, it is very nice. But the page title `Tutorials` might be changed.

**Comment by wxb** [comment]
@WanWizard 
Thanks, this page's URL is http://wxb.github.io/fuelPHP.zh-CN/index.html 
This document is based on the fuelphp 1.7-dev documents.
Later, I will gradually translation fuelphp 1.8-dev documentation. And I will put it on a dedicated Web site, for example, www.fuelphp.help
Thank you very much
